app/models/flashcard_set.rb

```diff
@@ -0,0 +1,3 @@
+class FlashcardSet < ApplicationRecord
+  belongs_to :teacher
```

Comment:
did you add has_many flashcard_sets to the teacher's model? it's not appearing here